AdminServer/tests/test_service_manager.py

```diff
@@ -0,0 +1,85 @@
+import io
+from collections import namedtuple
+
+from mock import MagicMock, patch
+from tornado.gen import Future
+from tornado.httpclient import AsyncHTTPClient
+from tornado.testing import AsyncTestCase, gen_test
+
+from appscale.admin.service_manager import (
+  DatastoreServer, gen, options, psutil, ServerStates, ServiceManager,
+  ServiceTypes)
+
+FakeHTTPResponse = namedtuple('Response', ['code'])
+
+# Skip sleep calls.
+sleep_response = Future()
+sleep_response.set_result(None)
+gen.sleep = MagicMock(return_value=sleep_response)
```

Comment:
Though I don't think this particular mock can affect any other tests, it is always safer to us `setUp*` and `tearDown*` function/methods for making sure that some real functions are mocked just for time test is running.
To patch something once for all test module you can use [setUpModule and tearDownModule functions](https://docs.python.org/2/library/unittest.html#setupmodule-and-teardownmodule).

So it may look like:
```
patchers = []

def setUpModule():
  patcher = patch.object(gen, 'sleep')
  patchers.append(patcher)
  sleep_response = Future()
  sleep_response.set_result(None)
  sleep_mock = patcher.start()
  sleep_mock.return_value = sleep_response

def tearDownModule():
  for patcher in patchers:
    patcher.stop()
```